src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossDF.scala

```diff
@@ -87,18 +91,45 @@ object TreeLossDF {
       sum("whrc_aboveground_co2_stock_2000__Mt") as "whrc_aboveground_co2_stock_2000__Mt"
     )
 
-    if (include) df
-    else {
-      df.groupBy(
+    (includePrimaryForest, includePlantations) match {
```

Comment:
Could you just make a list of group by columns, and add plantations and/or primary forest to it depending on these flags? That seems simpler than pattern matching all combinations and replicating the code 3 times

Reply:
yes, this makes more sense